### ISSUE TITLE
NameError: name 'exit' is not defined in sequences.pyx (fixed by adding import statement to the code) 

### ISSUE THREAD

**Issue by annashcherbina** [loosolab/TOBIAS#20]
In jupyter notebook, attempting to load the bias pickle: 
```
import pickle
filtered_atac_bias=pickle.load(open("K562.filtered_AtacBias.pickle",'rb'))
```
gives: 
```
NameError                                 Traceback (most recent call last)
<ipython-input-2-7942d1ef33fc> in <module>
      1 import pickle
      2 ## dnase, tobias, K562
----> 3 tobias_data_dnase_k562=pickle.load(open("/srv/scratch/annashch/bias_correction/enzymatic_bias/tobias/dnase/K562.filtered_AtacBias.pickle",'rb'))
      4 tobias_dnase_pssm_forward=np.transpose(tobias_data_dnase_k562.bias['forward'].pssm[0:4])[:,[0,2,3,1]]
      5 tobias_dnase_pssm_reverse=np.transpose(tobias_data_dnase_k562.bias['reverse'].pssm[0:4])[:,[0,2,3,1]]

~/TOBIAS/tobias/tools/atacorrect_functions.py in <module>
     28 
     29 #Internal functions and classes
---> 30 from tobias.utils.sequences import SequenceMatrix, GenomicSequence
     31 from tobias.utils.signals import fast_rolling_math
     32 from tobias.utils.utilities import *

~/TOBIAS/tobias/utils/sequences.pyx in init tobias.utils.sequences()

NameError: name 'exit' is not defined

```

This is fixed by adding an import statement in sequences.pyx 

```
from sys import exit 
```

**Comment by msbentsen** [comment]
Hi, ah yes I see, thanks for making me aware of this! I have fixed this in TOBIAS 0.11.3 (on PyPI now, and will be on bioconda once the bot updates it).

**Comment by annashcherbina** [comment]
Great, thank you!
